src/cmap/auth/mongodb_oidc/callback_workflow.ts

```diff
@@ -193,79 +132,28 @@ export class CallbackWorkflow implements Workflow {
    * Fetches an access token using either the request or refresh callbacks and
    * puts it in the cache.
    */
-  private async fetchAccessToken(
+  protected async fetchAccessToken(
     connection: Connection,
     credentials: MongoCredentials,
```

Comment:
```suggestion
    _connection: Connection,
    _credentials: MongoCredentials,
```

I'm surprised lint didn't complain about these variables being unused

Reply:
Updated.